.github/workflows/check-version-bumped.yml

```diff
@@ -0,0 +1,21 @@
+name: Check if PR has bumbed versions in Cargo.toml for code changes and if README is synchronized
```

Comment:
Seems a bit excessive for a workflow name. Perhaps change to "Check version bump" and explain in a comment what exactly it does.

I've been hoping there would be a `description` field better for this kind of information, but I couldn't find anything about this, so there probably isn't. :<

Reply:
Good point, since this name is displayed in UI like below:

![image](https://user-images.githubusercontent.com/3909333/149148830-224b455d-5bcb-4d4d-bd43-966fa4c208ff.png)

I'll shorten the name.